tests/instances.spec.ts

```diff
@@ -398,3 +398,15 @@ test("Migrate instance root storage volume to a different pool", async ({
   await migrateInstanceRootStorage(page, instance, "default", serverClustered);
   await deletePool(page, targetPool);
 });
+
+test("'Other' tab is removed when config/creating Instances, on LXD Version 5.0", async ({
+  page,
+  lxdVersion,
+}) => {
+  test.skip(
+    lxdVersion != "5.0-edge",
+    "Unavailable API Extensions: metadata_configuration",
```

Comment:
the reason why we would skip the test here is because `metadata_configuration` api extension exists in lxd 5.21 and latest/edge. Please also make the same change for the other test.

```suggestion
    "Newer LXD versions has the metadata_configuration API extension.",
```

Reply:
I think the skip message should say why the test is skipped. In my opinion, the current message is expressing that just correctly, no?

Reply:
I mean the message could be a bit more verbose and explain "In 5.0 we are missing the extension ... "

Reply:
Oh got you, yes, all right.

Reply:
I was trying to express the fact that 5.0 does NOT have the metadata_configuration, but I can replace it with the text that you provided, no worries.